packages/contracts/contracts/interfaces/IChugSplashManager.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+/**
+ * @title ChugSplashManager
+ * @notice Interface that must be inherited the ChugSplash manager.
+ */
+interface IChugSplashManager {
+    function activeBundleId() external returns (bytes32);
+
+    function proposers(address) external returns (bool);
+}
```

Comment:
I added these interfaces so that our foundry tests can just import them instead of the full contracts. This circumvents some issues I was encountering related to running the open zeppelin safety checks in the plugins package.